CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Added `autoformat-options` configuration option to let users specify command-line arguments to the Black formatting tool
 - Update `check_all` and `check_error` functions to let users pass in `typing.IO` objects to the `output` argument
+- Update the `forbidden-io-function-checker` to check functions from imported modules as well as methods (according to their qualified name)
```

Comment:
This is good; please also add a separate entry to record the enhancement that this checker now flags aliases of forbidden functions